lib/stripe/config.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Stripe.Config do
+  @moduledoc """
+  Utility that handles interaction with the application's configuration
+  """
+
+  @base_key :stripity_stripe
+
+  @doc """
+  Resolves the given key from the application's configuration returning the
+  wrapped expanded value. If the value was a function it get's evaluated, if
+  the value is a touple of three elements it gets applied.
+  """
+  @spec resolve(atom, any) :: any
+  def resolve(key, default \\ nil)
+  def resolve(key, default) when is_atom(key) do
+    Application.get_env(@base_key, key, default)
```

Comment:
`@base_key` could probably be inlined.

Reply:
You mean I should remove it and just write it as `Application.get_env(:stripety_stripe, key, default)` ?

Reply:
i think?  yeah

Reply:
Sry, a bit tired. I pushed the change, waiting for the CI server to give the green light.